driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/gridfs/GridFSPublisherSpecification.groovy

```diff
@@ -160,7 +160,7 @@ class GridFSPublisherSpecification extends FunctionalSpecification {
         subscriber = new ObservableSubscriber()
         publisher = gridFSBucket.downloadToPublisher(fileId as ObjectId).bufferSizeBytes(contentBytes.length * 3)
         publisher.subscribe(subscriber)
-        def data = subscriber.await(1, 30, SECONDS).getReceived().get(0)
+        def data = subscriber.await(1, 60, SECONDS).getReceived().get(0)
```

Comment:
It would be good to use `com.mongodb.ClusterFixture.TIMEOUT` here as well. I tried this in IntelliJ, and the symbol was not recognized, which is weird because `TIMEOUT` is used in other reactive files.

Reply:
Will update